sql/hive/compatibility/src/test/scala/org/apache/spark/sql/hive/execution/HiveCompatibilitySuite.scala

```diff
@@ -363,7 +363,11 @@ class HiveCompatibilitySuite extends HiveQueryFileTest with BeforeAndAfter {
     // Index commands are not supported
     "drop_index",
     "drop_index_removes_partition_dirs",
-    "alter_index"
+    "alter_index",
+
+    // We do not support macro for now.
+    // TODO: throw an exception if a user is trying to create a macro.
```

Comment:
This has been done in https://github.com/apache/spark/commit/c238cd07448f94bbceb661daad90b6a6d597a846